ports/stm32/mpbthciport.c

```diff
@@ -24,6 +24,7 @@
  * THE SOFTWARE.
  */
 
+#include <stdio.h>
```

Comment:
Is this still needed or left over from debugging?

Reply:
somewhat... it could be
``` c
// #include <stdio.h>
#define DEBUG_printf(...) // printf("mpbthciport.c: " __VA_ARGS__)
```

but yeah the DEBUG_printf doesn't work without it... a mild annoyance to no have it?

Reply:
Ah yeah I guess so. And we do this elsewhere too.